dev/start.bash

```diff
@@ -0,0 +1,5 @@
+kind create cluster
```

Comment:
should probably pick a unique cluster name to differentiate the armada-operator dev env